app/data/repositories/CardsRepository.py

```diff
@@ -0,0 +1,72 @@
+from app.data import dbConn
+from app.data.models.Card import Card
+
+
+conn = dbConn.get()
+
+
+def create(deck_id, word, translation):
+
+    deck_id = deck_id
```

Comment:
is deck_id on the left something else than deck_id on the right? if it is class member it would be nice to give it different name, check python convention for naming class members

Reply:
I noticed it and corrected it. done